smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/providers/connectors/ExecutionHolder.java

```diff
@@ -19,6 +24,8 @@
 @ApplicationScoped
 public class ExecutionHolder {
 
+    private static final String REACTIVE_MESSAGING_VERTX_CDI_QUALIFIER = "mp.messaging.connector.vertx.cdi.qualifier";
```

Comment:
Shouldn't we change the name of this property something like `mp.messaging.connector.vertx.cdi.identifier` ?

Reply:
done